app/controllers/partners/helps_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Partners
+  class HelpsController < BaseController
+    layout 'partners/application'
+
+    def show
+      @partner = current_partner
```

Comment:
Do we end up using this partner we've set in the view?

Reply:
Nope; good eye!